css/index.css

```diff
@@ -0,0 +1,43 @@
+@import "./navbar.css";
+
+.container-main__content {
+  width: 90%;
+}
+
+.main__heading {
+  font-size: 4rem;
+  font-family: var(--font-heading);
+  font-weight: bolder;
+}
+
+.categories__heading {
+  font-size: 2.5rem;
+  font-weight: bolder;
```

Comment:
try to use variables for font weight, bold bolder etc

Reply:
sure, i'll add these classes to my utility class.